src/Pages/Homepage/HomePage.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { useEffect } from 'react'
```

Comment:
imports could be done on one line for react

Reply:
Thank you done all the changes!